plugin/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/handle-prototype.js

```diff
@@ -1,8 +1,6 @@
 function handlePrototype(url) {
     buildFormTree(document.forms.config);
-    // TODO JSON.stringify fails in some circumstances: https://gist.github.com/jglick/70ec4b15c1f628fdf2e9 due to Array.prototype.toJSON
```

Comment:
(Confirmed that the bug described in the Gist is no longer reproducible in script console of either Firefox or Chrome.)